### DISCUSSION TITLE
hover:decoration-current not working

### DISCUSSION THREAD

**Discussion by thomasviaud** [tailwindlabs/tailwindcss#10524] [Help]
**What version of Tailwind CSS are you using?**

3.2.4

**What build tool (or framework if it abstracts the build tool) are you using?**

    "next": "13.1.6",
    "autoprefixer": "^10.4.13",
    "postcss": "^8.4.21",
    "tailwindcss": "^3.2.4"

**What version of Node.js are you using?**

v18

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**Describe your issue**

Hi, i'm trying to use the class "hover:decoration-current" on a button that is "decoration-transparent" by default and has a "text-white" class. It does not have any effect. If I'm using "hover:decoration-yellow" for instance, it works. When I'm doing my own css class (.link:hover { text-decoration-color: currentColor; }), the white color is applied as expected.

**Comment by adamwathan** [comment]
Hey! Here's a working demo of what it sounds like you're trying to do:

https://play.tailwindcss.com/A117sujqjJ
